packages/mui-material/src/Link/Link.js

```diff
@@ -60,6 +60,9 @@ const LinkRoot = styled(Typography, {
         textDecorationColor: 'inherit',
       },
     }),
+    '&:active': {
+      userSelect: 'none',
+    },
```

Comment:
Will we ever need this?

I can think of 3 types of elements we can render:

1. `<a>` solved by built-in behavior.
2. `<button> solved by built-in behavior.
3. regular element, e.g. `<div>`. WUT? doesn't make sense

```suggestion
```

Reply:
We will need this in Firefox to have the same behavior, as buttons in Firefox are not selectable by dragging from outside. We could add it inside the `ownerState.component === 'button'` condition, so it only applies to buttons, but I don't see the harm of supporting regular elements as well. What do you think?